framework-docs/modules/ROOT/pages/data-access/r2dbc.adoc

```diff
@@ -417,7 +417,7 @@ Do not pass `Collection<String>` or the like as an array parameter.
 [[r2dbc-DatabaseClient-filter]]
 ==== Statement Filters
 
-Sometimes it you need to fine-tune options on the actual `Statement`
+Sometimes if you need to fine-tune options on the actual `Statement`
```

Comment:
```suggestion
Sometimes you need to fine-tune options on the actual `Statement`
```

I think "it" should be completely removed, and the subsequent sentence should probably start with "To do so, register...".

Feel free to make that change in this PR. Otherwise I'll make the change when merging.